<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xdynamics</class>
 <widget class="QMainWindow" name="xdynamics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xdynamics</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="GraphicArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>990</width>
         <height>728</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="Import_MilkShape_3D_ASCII"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionMBD_Result_ASCII"/>
     <addaction name="actionDEM_Result_ASCII"/>
    </widget>
    <addaction name="actionChange_Shape"/>
    <addaction name="menuExport"/>
    <addaction name="menuImport"/>
    <addaction name="actionProperty"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChange_Shape">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Change Shape</string>
   </property>
  </action>
  <action name="Import_MilkShape_3D_ASCII">
   <property name="text">
    <string>MilkShape 3D ASCII</string>
   </property>
  </action>
  <action name="actionMBD_Result_ASCII">
   <property name="text">
    <string>MBD Result ASCII</string>
   </property>
  </action>
  <action name="actionDEM_Result_ASCII">
   <property name="text">
    <string>DEM Result ASCII</string>
   </property>
  </action>
  <action name="actionProperty">
   <property name="text">
    <string>Property</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="parview.qrc"/>
 </resources>
 <connections/>
</ui>
